src/utils/BaseService.ts

```
@@ -0,0 +1,44 @@
+  protected constructor(protected baseUrl: string) {}
+
+  protected async get<T>(url: string = ''): Promise<T> {
+    try {
+      const response = await api.get(`${this.baseUrl}${url}`)
+      return response.data
+    } catch (error) {
+      throw HttpErrorHandler.handle(error as AxiosError)
+    }
+  }
+
+  protected async post<T>(data: any, url: string = ''): Promise<T> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)